.test-infra/jenkins/job_PostCommit_Python37.groovy

```diff
@@ -27,7 +27,7 @@ PostcommitJobBuilder.postCommitJob('beam_PostCommit_Python37', 'Run Python 3.7 P
   previousNames('/beam_PostCommit_Python3_Verify/')
 
   // Set common parameters.
-  commonJobProperties.setTopLevelMainJobProperties(delegate)
+  commonJobProperties.setTopLevelMainJobProperties(delegate, 'master', 120)
```

Comment:
Does groovy allow to pass key-value pairs here, so that we ca avoid passing 'master'?